CHANGELOG.md

```diff
@@ -16,6 +16,70 @@
 
 ---
 
+## v3.0.0
+
+Here is the list of all the breaking changes between version 2 and 3. For each change, we show the new way to handle your use case.
```

Comment:
we should clarify that breaking changes are only for RUM SDK

Reply:
Done